Pipfile

```diff
@@ -16,3 +18,5 @@ types-requests = "*"
 [packages]
 requests = "*"
 camel-converter = {extras = ["pydantic"], version = "*"}
+isort = "*"
+black = "*"
```

Comment:
Hi @Dark-Rock,
Both dependencies should only be added to `[dev-packages]` not to `[packages]` that is why the `--dev` option is. Can you remove it?
Thank you 😊